tests/unattended/unit/suites/test-elasticsearch.sh

```diff
@@ -133,23 +129,22 @@ test-configureElasticsearch-dist-one-elastic-node() {
 
 
 test-configureElasticsearch-dist-one-elastic-node-assert() {
-    getConfig elasticsearch/elasticsearch_unattended_distributed.yml /etc/elasticsearch/elasticsearch.yml
     getConfig elasticsearch/roles/roles.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml
     getConfig elasticsearch/roles/roles_mapping.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml
     getConfig elasticsearch/roles/internal_users.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml
 
     rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f
```

Comment:
You can use rm /etc/elasticsearch/{name1,name2,...}`

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1188/commits/107118c34312b821527a77cfc9dfbc175bcb14e4